EU4toV2/Source/V2World/V2Reforms.h

```diff
@@ -47,17 +47,16 @@ class V2Reforms {
 		V2Reforms(const V2Country*, const std::shared_ptr<EU4::Country>);
 		void output(FILE*) const;
 	private:
-		void governmentEffects(const V2Country*);
-		void upperHouseEffects(const V2Country*);
-
-		double slavery;
-		double vote_franchise;
-		double upper_house_composition;
-		double voting_system;
-		double public_meetings;
-		double press_rights;
-		double trade_unions;
-		double political_parties;
+
+		bool abolishSlavery = false;
+		double slavery = 5.0;
+		double vote_franchise = 5.0;
+		double upper_house_composition = 5.0;
+		double voting_system = 5.0;
+		double public_meetings = 5.0;
+		double press_rights = 5.0;
+		double trade_unions = 5.0;
+		double political_parties = 5.0;
 };
 
 
```

Comment:
Should probably split V2UncivReforms into its own files.

Reply:
split files